src/main/java/com/coveros/selenified/application/Check.java

```diff
@@ -0,0 +1,830 @@
+/*
+ * Copyright 2019 Coveros, Inc.
+ *
+ * This file is part of Selenified.
+ *
+ * Selenified is licensed under the Apache License, Version
+ * 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy
+ * of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.coveros.selenified.application;
+
+import com.coveros.selenified.OutputFile;
+import com.coveros.selenified.OutputFile.Success;
+
+/**
+ * Assert will handle all verifications performed on the actual application
+ * itself. These asserts are custom to the framework, and in addition to
+ * providing easy object oriented capabilities, they take screenshots with each
+ * verification to provide additional traceability, and assist in
+ * troubleshooting and debugging failing tests.
+ *
+ * @author Max Saperstone
+ * @version 3.1.0
+ * @lastupdate 3/5/2019
+ */
+public interface Check {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Move constants to a class or enum. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS1214)